### PR TITLE
translation

### PR DESCRIPTION
**Pull request by lis102**
translate 2020-09-04-introducing-structured-logs.md into Chinese

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
@lis102
Pls sign the CLA in order for the PR to be reviewed.
ckeck
https://github.com/kubernetes/community/blob/master/CLA.md

**Comment by lis102** [comment]
I signed it

**Comment by lis102** [comment]
> @lis102
> Pls sign the CLA in order for the PR to be reviewed.
> ckeck
> https://github.com/kubernetes/community/blob/master/CLA.md

I signed it 10 hours ago but the status just doesn't change. Could you do me a favor?

**Comment by Arhell** [comment]
try re-uploading the commit

**Comment by lis102** [comment]
> /check-cla

image
Here is my status in linuxfoundation. It seems ok.

File: content/zh/blog/_posts/2020-09-04-introducing-structured-logs.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: blog
-title: 'Introducing Structured Logs'
+title: '介绍 Structured Logs'
```
Comment (tengqm) [inline content/zh/blog/_posts/2020-09-04-introducing-structured-logs.md:3]: 结构化日志介绍

**Comment by lis102** [comment]
> Thanks for your pull request. Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).
> 
> 📝 **Please follow instructions at https://git.k8s.io/community/CLA.md#the-contributor-license-agreement to sign the CLA.**
> 
> It may take a couple minutes for the CLA signature to be fully registered; after that, please reply here with a new comment and we'll verify. Thanks.
> 
> * If you've already signed a CLA, it's possible we don't have your GitHub username or you're using a different email address.  Check your existing CLA data and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> * If you signed the CLA as a corporation, please sign in with your organization's credentials at https://identity.linuxfoundation.org/projects/cncf to be authorized.
> * If you have done the above and are still having issues with the CLA being reported as unsigned, please log a ticket with the Linux Foundation Helpdesk: https://support.linuxfoundation.org/
> * Should you encounter any issues with the Linux Foundation Helpdesk, send a message to the backup e-mail support address at: [login-issues@jira.linuxfoundation.org](mailto:login-issues@jira.linuxfoundation.org)

I signed it

**Comment by lis102** [comment]
I signed it

**Comment by tengqm** [comment]
@lis102 You may want to check if you have committed the PR using the same email account you used for CLA. If not, you will need to check `git config -` and look for the `user.email` line. In case the `user.email` is not correct. You need to change the git config and then

```
git commit --amend --reset-author
git push -f
```

**Comment by Arhell** [comment]
Try to do everything in 1 commit.
I just see that the 1st one is @theonels
Since 3 commits
git reset --soft HEAD ~ 3
Then make a new commit and force push

**Pull request closed without merging**